src/HPolytope.jl

```diff
@@ -236,3 +238,10 @@ function vertices_list(P::HPolytope{N};
     prunefunc(P)
     return collect(points(P))
 end
+
+function linear_map(M::Matrix{MN}, P::HPolytope{SN}; kwargs...) where {MN<:Real, SN<:Real}
```

Comment:
Do we always require a `Matrix` (in constrast to `AbstractMatrix`)?

Reply:
I don't know if it's possible to make the numeric type conversion for *any* `AbstractMatrix` work, but i can try: if `promote_rule` has been defined by the type of `M` perhaps this just works..

Reply:
Broadcasting on the numeric type works without having to assume a specific matrix type, so ill do it that way...